plugins/org.jkiss.dbeaver.model.ai/src/org/jkiss/dbeaver/model/ai/speech/TranscriptResult.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2025 DBeaver Corp
+ *
+ * All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of DBeaver Corp and its suppliers, if any.
+ * The intellectual and technical concepts contained
+ * herein are proprietary to DBeaver Corp and its suppliers
+ * and may be covered by U.S. and Foreign Patents,
+ * patents in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from DBeaver Corp.
+ */
+package org.jkiss.dbeaver.model.ai.speech;
+
+public record TranscriptResult(String text, Usage usage) {
+
+    public record Usage(String type, int seconds) { }
```

Comment:
What's `type`? And please use `java.time.Duration duration` instead of `int seconds`